presto-kafka/src/main/java/io/prestosql/plugin/kafka/encoder/AbstractRowEncoder.java

```diff
@@ -18,6 +18,9 @@
 import com.google.common.primitives.SignedBytes;
```

Comment:
Commit message is a bit misleading. You didn't add support for structural types, but you allowed to implementations of `AbstractRowEncoder` to support them.

[optional] It would be nice use it, how about insert of structural types in Avro for Kafka?

Reply:
> [optional] It would be nice use it, how about insert of structural types in Avro for Kafka?

Will be raising a follow up PR for implementation in Avro and JSON